benches/oneshot/ipa.rs

```diff
@@ -0,0 +1,101 @@
+use rand::thread_rng;
+use raw_ipa::error::Error;
+use raw_ipa::ff::{Field, Fp32BitPrime};
+use raw_ipa::protocol::ipa::ipa;
+use raw_ipa::protocol::ipa::IPAInputRow;
+use raw_ipa::protocol::QueryId;
+use raw_ipa::secret_sharing::Replicated;
+use raw_ipa::test_fixture::{IntoShares, MaskedMatchKey, Runner, TestWorld, TestWorldConfig};
+use std::time::Instant;
+
+use rand::{distributions::Standard, prelude::Distribution, Rng};
+
+#[derive(Debug)]
+pub struct IPAInputTestRow {
+    match_key: u64,
+    is_trigger_bit: u128,
+    breakdown_key: u128,
+    trigger_value: u128,
+}
+
+impl<F> IntoShares<IPAInputRow<F>> for IPAInputTestRow
+where
+    F: Field + IntoShares<Replicated<F>>,
+    Standard: Distribution<F>,
+{
+    fn share_with<R: Rng>(self, rng: &mut R) -> [IPAInputRow<F>; 3] {
+        let match_key_shares = MaskedMatchKey::mask(self.match_key).share_with(rng);
+        let [itb0, itb1, itb2] = F::from(self.is_trigger_bit).share_with(rng);
+        let [bdk0, bdk1, bdk2] = F::from(self.breakdown_key).share_with(rng);
+        let [tv0, tv1, tv2] = F::from(self.trigger_value).share_with(rng);
```

Comment:
It makes me sad to see this code duplication. @akoshelev or @martinthomson - any ideas on how to avoid this?

Reply:
I couldnt import from the test modules even after making them public. It would be great to have test modules exposed if it is possible